r/tools/nixlibs.R

```diff
@@ -371,9 +371,17 @@ ensure_cmake <- function() {
   if (is.null(cmake)) {
     # If not found, download it
     cat("**** cmake\n")
-    CMAKE_VERSION <- Sys.getenv("CMAKE_VERSION", "3.19.2")
+    CMAKE_VERSION <- Sys.getenv("CMAKE_VERSION", "3.19.3")
     if (tolower(Sys.info()[["sysname"]]) %in% "darwin") {
       postfix <- "-macos-universal.tar.gz"
+    } else if (tolower(Sys.info()[["machine"]]) == "arm64") {
+      postfix <- "-linux-aarch64.tar.gz"
+    } else if (tolower(Sys.info()[["machine"]]) == "armv7l") { # e.g. Raspberry Pi OS
+      # It is unlikely that we would hit this logic as cmake is usual installed with Pi OS
+      stop(paste0(
+         "*** cmake was not found locally and no binaries are available for your architecture - it must be compiled from source",
```

Comment:
Is this the case or can we just point to the source directory here like we have with the binaries?

Reply:
Let's say less, just so CRAN doesn't get the right idea. 

```suggestion
         "*** cmake not available",
```